l1-contracts/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -210,18 +221,18 @@ contract L1ERC20Bridge is IL1ERC20Bridge, ReentrancyGuard {
         uint256 _l2TxGasPerPubdataByte,
         address _refundRecipient
     ) public payable nonReentrant returns (bytes32 l2TxHash) {
-        // empty deposit
         if (_amount == 0) {
+            // empty deposit amount
             revert EmptyDeposit();
         }
-        uint256 amount = _depositFundsToSharedBridge(msg.sender, IERC20(_l1Token), _amount);
-        // The token has non-standard transfer logic
+        uint256 amount = _depositFundsToNTV(msg.sender, IERC20(_l1Token), _amount);
```

Comment:
Probably yes, we did not test it. 
Alternatively we deposited to AR, and then transferred to NTV, I'm not sure.

Reply:
Probably this is wrong, changing it back

Reply:
It is wrong because the way the old SDK works is that we deposit to the SharedBridge = AR, and then the NTV withdraws either from it or from the sender. So we have to transfer to the AR.